### PR TITLE
EXEMPLE OF PR: How to do a pr

### PR DESCRIPTION
**Pull request by BarnabePILLIAUDIN**
# IMPORTANT
THIS IS AN EXEMPLE TO EXPLAIN PR DON'T MERGE IT !

# DONE 
- Some bullshit to explain how a pr and a cr work

# TO TEST

- You should see a file named toto.txt with "some bullshit to explain pr and code reviews" in it

### CODE REVIEW

File: toto.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+some bullshit to explain pr and code reviews
```
Comment (cedricgautier) [inline toto.txt:1]: Remove "bullshit"

Comment (cedricgautier) [inline toto.txt:1]: Capitalize "Some"

**Review by cedricgautier — DISMISSED** [review]
LGTM!

**Comment by KyDev25** [comment]
Add the word "test"

File: toto.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Some dummy text to explain pr and code reviews
```
Comment (KyDev25) [inline toto.txt:1]: Add the word "test"

```suggestion
Some dummy text to explain pr and code reviews test
```

**Review by KyDev25 — CHANGES_REQUESTED** [review]
Do it

**Review by KyDev25 — APPROVED** [review]
OK

**Review by cedricgautier — APPROVED** [review]

**Pull request closed without merging**